src/store/state/state-model.ts

```diff
@@ -1,5 +1,9 @@
 import { StateModelIF } from '@/interfaces'
 
 export const stateModel: StateModelIF = {
-  stateText: 'Base State Text'
+  stateText: 'Base State Text',
+  currentStep: 1,
+  isSaving: false,
+  isSavingResuming: false,
+  isFilingPaying: false
```

Comment:
I mentioned something very similar in Lekshmi's PR. This state model was initially just an example model. Would it make sense to group some of the properties in their own model? Also we could use this as a generic model, if you feel the data is just that. In that case, we should also remove the example property `stateText` 👍

Reply:
I think it doesn't make sense to create other models yet, with just this data.

But soon, we will have more data (for each step) (and also page validation state). At that point, yes, it makes sense to group that data into its own model as needed.

Reply:
Sounds good!